doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>gradient based -&gt; gradient-based</p><p>That’s it, similarity quantifies how much an additional training step on&nbsp;z would change the loss calculated at&nbsp;z'</p><p>-&gt; not sure about the start of the sentence "That's it" -- maybe not the best choice of words</p><p>-&gt; I think that training step might be misinterpreted in this context as it is not really a training step. Maybe be a bit more rigorous about it.</p><p>consider the Taylor expansion -&gt; consider the first-order Taylor expansion (everywhere)</p><p>by an amount \epsilon - still not rendering properly</p><p>That's it, the kernel -- maybe it's just me, but I don't know how to read "That's it, ". Is it an exclamation "That's it!"</p><p>add some tags to the symmetric kernels</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I'm sure you mean "that is"

Reply:
<p>I rephrase "&nbsp;I think that training step might be misinterpreted in this context as it is not really a training step. Maybe be a bit more rigorous about it."</p><p> </p><p> </p>

Reply:
<p>I always used that is it ... </p>

Reply:
<p>In any case, I substituted it with "in conclusion"</p>

Reply:
@gipster there's another case of this under "Theory"